connectors/standalone/src/main/java/io/delta/standalone/expressions/Like.java

```diff
@@ -0,0 +1,31 @@
+package io.delta.standalone.expressions;
+
+/**
+ * Evaluates {@code expr1} like {@code expr2} for {@code new Like(expr1, expr2)}.
+ */
+public class Like extends BinaryOperator implements Predicate {
+    public Like(Expression left, Expression right) {
+        super(left, right, "like");
+    }
+
+    @Override
+    protected Object nullSafeEval(Object leftResult, Object rightResult) {
+        String str = String.valueOf(rightResult);
+        String subStr;
+        if (str.startsWith("%")) {
```

Comment:
does this work, if you have `%` or `_` in the middle of the string? Probably it is easier if you change the `%` and `_` to java regex equivalent and just do java regex.